README.md

```diff
@@ -13,36 +13,36 @@ Harvester is an open source [hyper-converged infrastructure](https://en.wikipedi
+1. Supported VM backup, snapshot and restore
+1. Distributed block storage and storage tiering
+1. Multiple network interface controllers (NICs) in the VM connecting to the management network or VLANs
 1. Virtual Machine and cloud-init templates
-1. [Rancher integration](https://docs.harvesterhci.io/v1.0/rancher/rancher-integration/) with multi-cluster management and the Harvester node driver
-1. [PXE/iPXE boot support](https://docs.harvesterhci.io/v1.0/install/pxe-boot-install)
+1. [Rancher](https://github.com/rancher/rancher) integration with multi-cluster management and the Harvester node driver
+1. [PXE/iPXE boot support](https://docs.harvesterhci.io/latest/install/pxe-boot-install)
 1. Virtual IP and bond NIC support
-1. Monitoring integration
+1. Monitoring and logging integration
 
 The following diagram outlines a high-level architecture of Harvester:
 
 ![](./docs/assets/architecture.svg)
 
 - [Longhorn](https://longhorn.io/) is a lightweight, reliable, and easy-to-use distributed block storage system for Kubernetes.
 - [KubeVirt](https://kubevirt.io/) is a virtual machine management add-on for Kubernetes.
-- [Elemental for openSUSE Leap 15.3](https://github.com/rancher-sandbox/cOS-toolkit) is a Linux distribution designed to remove as much OS maintenance as possible in a Kubernetes cluster.
+- [Elemental for SLE-Micro 5.2](https://github.com/rancher-sandbox/cOS-toolkit) (based on openSUSE Leap 15.3 before v1.0.3) is an immutable Linux distribution designed to remove as much OS maintenance as possible in a Kubernetes cluster.
 
 ## Hardware Requirements
 To get the Harvester server up and running the following minimum hardware is required:
 
 | Type | Requirements |
 |:---|:---|
-| CPU | x86_64 only. Hardware-assisted virtualization is required. 8-core processor minimum; 16-core or above preferred |
-| Memory | 32 GB minimum, 64 GB or above preferred |
-| Disk Capacity |  140 GB minimum, 500 GB or above preferred |
-| Disk Performance |  5,000+ random IOPS per disk(SSD/NVMe). Management nodes (first 3 nodes) must be [fast enough for Etcd](https://www.ibm.com/cloud/blog/using-fio-to-tell-whether-your-storage-is-fast-enough-for-etcd). |
-| Network Card | 1 Gbps Ethernet minimum, 10Gbps Ethernet recommended |
-| Network Switch | Trunking of ports required for VLAN support |
+| CPU | x86_64 only. Hardware-assisted virtualization is required. 8-core processor minimum for testing; 16-core or above preferred for production                                                                |
+| Memory | 32 GB minimum; 64 GB or above preferred                                                                                                                                                                    |
+| Disk Capacity | 140 GB minimum for testing; 500 GB or above preferred for production                                                                                                                                       |
```

Comment:
200 GB minimum is required for testing upgrade. Do we want to update here?

Reply:
done, thanks.